examples/run_CPMG.py

```diff
@@ -66,56 +61,54 @@
 )
 # }}}
 # {{{run cpmg
-# NOTE: Number of segments is nEchoes * nPhaseSteps
-data = run_cpmg(
-    nScans = config_dict["nScans"],
-    indirect_idx = 0,
-    indirect_len = 1,
-    ph1_cyc = ph1_cyc,
-    adcOffset = config_dict["adc_offset"],
-    carrierFreq_MHz = config_dict["carrierFreq_MHz"],
-    nPoints = nPoints,
-    nEchoes = config_dict["nEchoes"],
-    p90_us = config_dict["p90_us"],
-    repetition_us = config_dict["repetition_us"],
-    tau_us = config_dict["tau_us"],
-    SW_kHz = config_dict["SW_kHz"],
-    pad_start_us = pad_start,
-    pad_end_us = pad_end,
-    ret_data = None,
-)
+data = generic(
+        ppg_list = [
+                ("phase_reset", 1),
+                ("delay_TTL", config_dict['deblank_us']),
+                ("pulse_TTL", config_dict['p90_us'], "ph1", ph1_cyc),
+                ("delay", tau_us),
+                ("delay_TTL", config_dict['deblank_us']),
+                ("pulse_TTL", 2.0 * config_dict['p90_us'], "ph2", ph2_cyc),
```

Comment:
because we are doing our cycling manually, above, you want to *also* call this "ph1" (see the docstring for `apply_cycles`)

Reply:
actually,  to avoid confusion, I would just call this `ph_cycle`.'

and, since this is going into examples in the docstring on the top, you want to reference :func:`apply_cycles`

Reply:
mentioned in docstring